samples/bluetooth/peripheral/src/main.c

```diff
@@ -337,19 +337,34 @@ static void bas_notify(void)
 	bt_bas_set_battery_level(battery_level);
 }
 
+static uint8_t bt_heartrate = 90U;
```

Comment:
Minor suggestion: Suggest to introduce a `#define HEARTRATE_DEFAULT 90U` and use that instead of 3 literal ints